File: Programming-languages-and-laboratory/Python/Exercises/01_compito_java_20150119/Fuoristrada.py
from Automobile import Automobile


class Fuoristrada(Automobile):
    def __init__(self, tipo, modello, produttore, marce, peso, codice_identificativo):
        super().__init__(tipo, modello, produttore, peso, codice_identificativo)
        self._marce = marce

    def __str__(self):
        return super().__str__() + " - " + str(self._marce)

File: Programming-languages-and-laboratory/Java/Exercises/01_compito_java_2015_01_19/Fuoristrada.java
public class Fuoristrada extends Auto {
    int marce;

    public Fuoristrada(String m, String pr, int p, int cod, int mar) {
        super(m, pr, p, cod);
        marce = mar;
    }

    public String toString() {
        return codice + "\t" + produttore + "\t" + modello + "\t-\t" + marce;
    }
}